docs/troubleshooting_installation.md

```diff
@@ -10,4 +10,7 @@
 ## Problem/Fix
 ### Install fails to create OVOS wifi hotspot 
 There is a known issue with balena (the wifi access point app) when it detects a WPA3 network of any sort, it fails.
-[More Information](https://github.com/balena-os/wifi-connect/issues/416)
\ No newline at end of file
+[More Information](https://github.com/balena-os/wifi-connect/issues/416)
+
+### Raspbian OVOS stuck on "Generating SSH keys..."
```

Comment:
This has been fixed.  It was an issue with pi-gen upstream

Reply:
oops, pushed the button at same time